lib/matplotlib/cbook/__init__.py

```diff
@@ -138,20 +138,14 @@ class CallbackRegistry:
     Parameters
     ----------
     exception_handler : callable, optional
-       If provided must have signature ::
-
-          def handler(exc: Exception) -> None:
-
-       If not None this function will be called with any `Exception`
-       subclass raised by the callbacks in `CallbackRegistry.process`.
-       The handler may either consume the exception or re-raise.
-
-       The callable must be pickle-able.
```

Comment:
So we're not inherently using the "pickle-able" nature to make the whole registry work, right?

Reply:
We don't, except if we pickle the CR.